modules/apps/27-interchain-accounts/module.go

```diff
@@ -219,11 +219,11 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 }
 
 // WeightedOperations is unimplemented.
-func (am AppModule) WeightedOperations(_ module.SimulationState) []simtypes.WeightedOperation {
+func (AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
```

Comment:
should we require names for variables not being used? I would have thought the underscore is okay here?

Reply:
and generally preferred, at least for me, w/o the underscore I spend some time looking around for usages.

Reply:
@DimitrisJim agree with you.